### PR TITLE
EIP-4907: Fixed typos

### PR DESCRIPTION
**Pull request by 0fatih**

### CODE REVIEW

**Comment by Pandapip1** [comment]
eipw errors need to be fixed

**Review by 0xanders — DISMISSED** [review]

**Comment by 0fatih** [comment]
@Pandapip1 Bot complains about the title:

```
3 | title: Rental NFT, ERC-721 User And Expires Extension
  |       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ too long
```

I didn't even touch the title. Should I change it?

**Comment by 0xanders** [comment]
Suggest  change the title to : Rental NFT, An Extension of EIP-721

**Comment by 0fatih** [comment]
Are these requirements new?

```
Error: proposals must be referenced with the form `EIP-N` (not `ERC-N`)
Error: proposals must be referenced with the form `EIP-N` (not `ERC-N`)
Error: proposals must be referenced with the form `EIP-N` (not `ERC-N`)
Error: proposals must be referenced with the form `EIP-N` (not `ERC-N`)
Error: proposals must be referenced with the form `EIP-N` (not `ERC-N`)
Error: proposals must be referenced with the form `EIP-N` (not `ERC-N`)
Error: proposals must be referenced with the form `EIP-N` (not `ERC-N`)
Error: proposals must be referenced with the form `EIP-N` (not `ERC-N`)
Error: validation found errors :(
```
Guys this is my first PR to EIPs. I wanted to start with the basics (typos). What is going on? What should I do?

File: EIPS/eip-4907.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 eip: 4907
-title: Rental NFT, ERC-721 User And Expires Extension
+title: Rental NFT, an Extension of EIP-721
 description: Add a time-limited role with restricted permissions to ERC-721 tokens.
```
Comment (0xanders) [inline EIPS/eip-4907.md:4]: description: Add a time-limited role with restricted permissions to EIP-721 tokens.

**Comment by Pandapip1** [comment]
Yes, those are requirements. Just replace all instances of `ERC-X` with `EIP-X`.

File: EIPS/eip-4907.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 eip: 4907
-title: Rental NFT, ERC-721 User And Expires Extension
+title: Rental NFT, an Extension of EIP-721
```
Comment (Pandapip1) [inline EIPS/eip-4907.md:3]:
```suggestion
title: Rental Extension of EIP-721
```

**Review by 0xanders — CHANGES_REQUESTED** [review]
line 4:  change ERC-721 to EIP-721

line 31: add \` to  [IERC4907.sol] ,  [`IERC4907.sol`]

line 90: change ERC-721 to EIP-721

line 92: change ERC-721 to EIP-721

line 97: add \` to  [ERC4907Demo.sol] ,  [`ERC4907Demo.sol`]

line 167: add \` to  [ERC4907.sol] ,  [`ERC4907.sol`]

**Comment by 0xanders** [comment]
IERC4907.sol, ERC4907.sol , ERC4907Demo.sol are file names,  you can add \` to them , such as [`IERC4907.sol`]

File: EIPS/eip-4907.md [hunk 3]
```diff
@@ -87,14 +87,14 @@ In the spirit of permission less interoperability, this standard makes it easier
...
+As mentioned in the specifications section, this standard can be fully EIP-721 compatible by adding an extension function set.
 
-In addition, new functions introduced in this standard have many similarities with the existing functions in ERC-721. This allows developers to easily adopt the standard quickly.
+In addition, new functions introduced in this standard have many similarities with the existing functions in EIP-721. This allows developers to easily adopt the standard quickly.
 
 ## Test Cases
 
 ### Test Contract
-ERC4907Demo Implementation: [ERC4907Demo.sol](../assets/eip-4907/contracts/ERC4907Demo.sol)
+EIP4907Demo Implementation: [EIP4907Demo.sol](../assets/eip-4907/contracts/ERC4907Demo.sol)
```
Comment (0xanders) [inline EIPS/eip-4907.md]: `ERC4907Demo` Implementation: [`ERC4907Demo.sol`](../assets/eip-4907/contracts/ERC4907Demo.sol)

File: EIPS/eip-4907.md [hunk 4]
```diff
@@ -164,7 +164,7 @@ truffle test ./test/test.js
 ```
 
 ## Reference Implementation
-ERC4907 Implementation: [ERC4907.sol](../assets/eip-4907/contracts/ERC4907.sol)
+EIP4907 Implementation: [ERC4907.sol](../assets/eip-4907/contracts/ERC4907.sol)
```
Comment (0xanders) [inline EIPS/eip-4907.md]: EIP-4907 Implementation: [`ERC4907.sol`](../assets/eip-4907/contracts/ERC4907.sol)

**Review by 0xanders — CHANGES_REQUESTED** [review]

File: EIPS/eip-4907.md [hunk 5]
```diff
@@ -164,7 +164,7 @@ truffle test ./test/test.js
 ```
 
 ## Reference Implementation
-ERC4907 Implementation: [ERC4907.sol](../assets/eip-4907/contracts/ERC4907.sol)
+EIP-4907 Implementation: [`ERC4907.sol`](../assets/eip-4907/contracts/ERC4907.sol)
```
Comment (0xanders) [inline EIPS/eip-4907.md:167]: Implementation: [`ERC4907.sol`](../assets/eip-4907/contracts/ERC4907.sol)

File: EIPS/eip-4907.md [hunk 6]
```diff
@@ -87,14 +87,14 @@ In the spirit of permission less interoperability, this standard makes it easier
...
+As mentioned in the specifications section, this standard can be fully EIP-721 compatible by adding an extension function set.
 
-In addition, new functions introduced in this standard have many similarities with the existing functions in ERC-721. This allows developers to easily adopt the standard quickly.
+In addition, new functions introduced in this standard have many similarities with the existing functions in EIP-721. This allows developers to easily adopt the standard quickly.
 
 ## Test Cases
 
 ### Test Contract
-ERC4907Demo Implementation: [ERC4907Demo.sol](../assets/eip-4907/contracts/ERC4907Demo.sol)
+EIP-4907Demo Implementation: [`EIP4907Demo.sol`](../assets/eip-4907/contracts/ERC4907Demo.sol)
```
Comment (0xanders) [inline EIPS/eip-4907.md:97]: Test contract implementation: [`ERC4907Demo.sol`](../assets/eip-4907/contracts/ERC4907Demo.sol)

**Review by 0xanders — CHANGES_REQUESTED** [review]

**Review by 0xanders — APPROVED** [review]

File: EIPS/eip-4907.md [hunk 7]
```diff
@@ -87,14 +87,14 @@ In the spirit of permission less interoperability, this standard makes it easier
...
+As mentioned in the specifications section, this standard can be fully EIP-721 compatible by adding an extension function set.
 
-In addition, new functions introduced in this standard have many similarities with the existing functions in ERC-721. This allows developers to easily adopt the standard quickly.
+In addition, new functions introduced in this standard have many similarities with the existing functions in EIP-721. This allows developers to easily adopt the standard quickly.
 
 ## Test Cases
 
 ### Test Contract
-ERC4907Demo Implementation: [ERC4907Demo.sol](../assets/eip-4907/contracts/ERC4907Demo.sol)
+`EIP-4907Demo` Implementation: [`EIP4907Demo.sol`](../assets/eip-4907/contracts/ERC4907Demo.sol)
```
Comment (Pandapip1) [inline EIPS/eip-4907.md:97]:
```suggestion
`ERC4907Demo` Implementation: [`ERC4907Demo.sol`](../assets/eip-4907/contracts/ERC4907Demo.sol)
```

File: EIPS/eip-4907.md [hunk 8]
```diff
@@ -164,7 +164,7 @@ truffle test ./test/test.js
 ```
 
 ## Reference Implementation
-ERC4907 Implementation: [ERC4907.sol](../assets/eip-4907/contracts/ERC4907.sol)
+`EIP-4907` Implementation: [`ERC4907.sol`](../assets/eip-4907/contracts/ERC4907.sol)
```
Comment (Pandapip1) [inline EIPS/eip-4907.md:167]:
```suggestion
Implementation: [`ERC4907.sol`](../assets/eip-4907/contracts/ERC4907.sol)
```
Avoid self-references where possible.

**Review by Pandapip1 — CHANGES_REQUESTED** [review]

**Review by Pandapip1 — DISMISSED** [review]

**Comment by Pandapip1** [comment]
Needs approval by @0xanders, or @LanceSnow

File: EIPS/eip-4907.md [hunk 9]
```diff
@@ -28,17 +28,17 @@ Furthermore, applications of this model (such as renting) often demand that user
 The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY" and "OPTIONAL" in this document are to be interpreted as described in RFC 2119.
 
 ### Contract Interface
-Solidity Interface with NatSpec & OpenZeppelin v4 Interfaces (also available at [IERC4907.sol](../assets/eip-4907/contracts/IERC4907.sol)):
+Solidity Interface with NatSpec & OpenZeppelin v4 Interfaces (also available at [`IEIP4907.sol`](../assets/eip-4907/contracts/IERC4907.sol)):
```
Comment (0xanders) [inline EIPS/eip-4907.md:31]: line31:  change  `IEIP4907.sol`  to `IERC4907.sol`

File: EIPS/eip-4907.md (hunk 7 above)
Comment (0xanders) [inline EIPS/eip-4907.md:97]: line97:  change  `IEIP4907Demo.sol`  to `ERC4907Demo.sol`

**Review by 0xanders — DISMISSED** [review]

File: EIPS/eip-4907.md [hunk 10]
```diff
@@ -87,14 +87,14 @@ In the spirit of permission less interoperability, this standard makes it easier
...
+As mentioned in the specifications section, this standard can be fully EIP-721 compatible by adding an extension function set.
 
-In addition, new functions introduced in this standard have many similarities with the existing functions in ERC-721. This allows developers to easily adopt the standard quickly.
+In addition, new functions introduced in this standard have many similarities with the existing functions in EIP-721. This allows developers to easily adopt the standard quickly.
 
 ## Test Cases
 
 ### Test Contract
-ERC4907Demo Implementation: [ERC4907Demo.sol](../assets/eip-4907/contracts/ERC4907Demo.sol)
+`EIP-4907Demo` Implementation: [`ERC4907Demo.sol`](../assets/eip-4907/contracts/ERC4907Demo.sol)
```
Comment (Pandapip1) [inline EIPS/eip-4907.md:97]:
```suggestion
`ERC4907Demo` Implementation: [`ERC4907Demo.sol`](../assets/eip-4907/contracts/ERC4907Demo.sol)
```

**Review by Pandapip1 — APPROVED** [review]

**Review by 0xanders — APPROVED** [review]

**Pull request merged**